src/main/scala/mesosphere/marathon/storage/repository/GcActor.scala

```diff
@@ -69,7 +70,8 @@ private[storage] class GcActor[K, C, S](
     val appRepository: AppRepositoryImpl[K, C, S],
     val podRepository: PodRepositoryImpl[K, C, S],
     val maxVersions: Int,
-    val scanBatchSize: Int = 32)(implicit val mat: Materializer, val ctx: ExecutionContext)
+    val scanBatchSize: Int = 32,
+    val restingTimeout: FiniteDuration)(implicit val mat: Materializer, val ctx: ExecutionContext)
```

Comment:
Naming should stay consistent if possible. E.g. here too the parameter should be called smth. like `scanInterval`